### PR TITLE
Assets - Fixed images PR

### PR DESCRIPTION
**Pull request by wchrapcynski**
Per issue #136 fixed some issues with white space and stylized images.

### CODE REVIEW

**Comment by ZakkMan** [comment]
@wchrapcynski Can you update this to fix any merge conflicts?

File: client/src/App.js [hunk 1]
```diff
@@ -1,27 +1,26 @@
...
+          rel="noopener noreferrer"
+        >
+          Learn React
+        </a>
+      </header>
+    </div>
+  );
 }
 
 export default App;
```
Comment (ZakkMan) [inline client/src/App.js]: Looks like you accidentally deleted all of Tory's hard work!

Comment (wchrapcynski) [reply client/src/App.js]: Ahh, I thought I had just removed the default code put in when the App was created.

**Comment by wchrapcynski** [comment]
@ZakkMan Should be good for merging now.

**Pull request merged**